tests/base/fixtures/cart.page.ts

```diff
@@ -1,18 +1,63 @@
 import {expect, type Locator, type Page} from '@playwright/test';
```

Comment:
@shayfaber Let's keep the naming everywhere 'coupon code' as Magento is also naming it that way.